addons/hr/views/mail_activity_plan_views.xml

```diff
@@ -46,6 +62,12 @@
             <field name="res_model">mail.activity.plan</field>
             <field name="view_mode">tree,form</field>
             <field name="search_view_id" ref="mail.mail_activity_plan_view_search"/>
+            <!-- Force the hr view which do the activity template edition in a popup. -->
+            <field name="view_ids"
```

Comment:
It is better to define ir.actions.act_window.view records, as it eases modification, maintenance, update, ...

Reply:
Yes, good idea, indeed they can be changed independently so